manifest_template.yml

```diff
@@ -2,29 +2,36 @@ SERVICE_NAME: e-referrals-service-patient-care-api
 PRODUCT_DISPLAY_NAME: e-referrals-service-patient-care-api
 DESCRIPTION: e-RS Patient Care API
 
-# The request_limit and burst_limit variables included here define the ratelimiting applied to each policy. 
-# The request_limit variable defines how many requests can be made to the referenced proxy per minute, 
+# The request_limit and burst_limit variables included here define the ratelimiting applied to each policy.
+# The request_limit variable defines how many requests can be made to the referenced proxy per minute,
 # whereas the burst_limit defines how many requests can be sent to the reference proxy as a spike of traffic.
 APIGEE_ENVIRONMENTS:
     - name: internal-dev
+      approval_type: manual
```

Comment:
Given we always want manual approval for every environment (and presumably any new env that may be created) maybe worth removing this from env level config and just hard code to manual in the {{products.approvalType}} (x2)?